reflred/refldata.py

```diff
@@ -796,6 +796,9 @@ class ReflData(Group):
     mask = None
     #: Computed 1-sigma angular resolution in degrees
     angular_resolution = None  # type: np.ndarray
+    #: Source divergence (for systems with e.g. mirror optics)
+    #: as a full-width in degrees
+    source_divergence = None
```

Comment:
Please call this `source_divergence_fwhm` or switch to 1-σ.

Reply:
It's not really FWHM either - we can use 1-sigma but it's not right.

It's the width of a rectangular distribution.  Won't it be more confusing to make users try to figure out what 1-sigma for that would be?

Reply:
How about ```source_divergence_fullwidth```?

Reply:
FWHM for a rectangular distribution is the width of the rectangular distribution, so I don't see the confusion.

The main point of the request is that any divergence that is not in degrees 1-σ needs to be tagged in the name. I won't argue too much about what name that is so long as it is used consistently.

Do we need a distribution object which carries shape information instead of trying to capture them all in a single constant and a bunch of assumptions in the code?

Reply:
Using 1-sigma for a distribution that isn't Gaussian is confusing - the conversion from full-width to 1-sigma has a factor of 
 sqrt(ln(256)) everywhere *except here*.

Reply:
I use a conversion of Δt²/12 for variance in scale.py because uncertainty is uniform in a time step, so not quite everywhere ;-) But I see your point: I don't call it FWHM in that case.

If you don't know the distribution then square root of variance is a good measure of its width. That's why I'm using the Gaussian equivalent 1-σ internally. I'm concerned that future me will see `source_divergence` and use its value for mixing with other distribution without first converting it to the equivalent Gaussian. I would probably do the same here and let the function make assumptions about the distribution shape, but I'm not the one writing the code.

So assuming that source divergence is always and only uniform how about `source_divergence_uniform`? I find `source_divergence_fullwidth` to be ambiguous. It applies equally to triangular, and with an additional parameter to trapezoidal.